torch/_torch_docs.py

```diff
@@ -993,60 +993,71 @@ def merge_dicts(*dicts):
            r"""
 asarray(obj, *, dtype=None, device=None, copy=None, requires_grad=False) -> Tensor
 
-Converts :attr:`obj` into a tensor, sharing data and preserving autograd history
-if possible.
+Converts :attr:`obj` in a tensor.
 
 :attr:`obj` can be one of:
 
 1. a tensor
 2. a NumPy array
 3. a DLPack capsule
-4. a Python object that implements the buffer protocol
-5. a Python sequence
+4. an object that implements Python's buffer protocol
+5. a sequence
```

Comment:
does this just mean an ordered set?

Reply:
https://docs.python.org/3/library/stdtypes.html#sequence-types-list-tuple-range

This may be incorrect if we don't handle ranges. If we don't, however, we should update sequence to be "list" or "tuple" explicitly.

Reply:
Oops. This looks like my fault, here. It should read "a scalar".
But, regarding ranges, I think we do support them.
```python
>>> torch.tensor(range(5))
tensor([0, 1, 2, 3, 4])
>>> torch.asarray(range(5))
tensor([0, 1, 2, 3, 4])
```